cmd/infra/azure/create.go

```diff
@@ -824,32 +824,32 @@ func assignServicePrincipalRoles(subscriptionID, managedResourceGroupName, nsgRe
 	vnetRG := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", subscriptionID, vnetResourceGroupName)
 	dnsZoneRG := fmt.Sprintf("/subscriptions/%s/resourceGroups/%s", subscriptionID, dnsZoneResourceGroupName)
 
-	role := "Contributor"
+	role := "b24988ac-6180-42a0-ab88-20f7382dd24c"
 
 	scopes := []string{managedRG}
 
 	// TODO CNTRLPLANE-171: CPO, KMS, and NodePoolManagement will need new roles that do not exist today
 	switch component {
 	case cloudProvider:
-		role = "Azure Red Hat OpenShift Cloud Controller Manager Role"
+		role = "5b7237c5-45e1-49d6-bc18-a1f62f400748"
```

Comment:
I don't think this is right. It's the same as the azure disk and they were not the same as before.

Reply:
I fixed this. This should fix the failing e2e-aks test as well 🤞🏻 .